docs/src/parameter_distributions.md

```diff
@@ -1,13 +1,18 @@
 # Prior distributions
 
-We provide a flexible setup for storing prior distribution with the `ParameterDistributions` module found in `src/ParameterDistributions.jl`.
+Bayesian inference begins with an explicit prior distribution. This page describes the interface EnsembleKalmanProcesses provides for specifying priors on parameters, via the ParameterDistributions module (`src/ParameterDistributions.jl`).
 
-One can create a full parameter distribution using three inputs:
- 1. A distribution, given as a `ParameterDistributionType` object,
- 2. A constraint, or array of constraints, given as a `ConstraintType` or `Array{ConstraintType}` object,
- 3. A name, given as a `String`.
+## Summary
 
-In complex settings, one should build a `ParameterDistribution`-per-distribution, and then combine together. 
+A prior is specified by a `ParameterDistribution` object, which has three components:
+
+ 1. The distribution itself, given as a [ParameterDistributionType](@ref) object. This includes standard Julia [Distributions](https://juliastats.org/Distributions.jl/stable/) as well as empirical/sample-based distributions, and can be uni- or multi-variate. To clarify, despite our use of the term "Kalman processes," the prior distribution is *not* required to be Gaussian.
+ 2. A constraint (or array of constraints) on the domain of the distribution, given as a [ConstraintType](@ref) or `Array{ConstraintType}` object (the latter case builds a multivariate constraint as the Cartesian product of one-dimensional Constraints). This is used to enforce physical parameter values during inference: the model is never evaluated at parameter values outside the constrained region, and the posterior distribution will only be supported there.
+ 3. The parameter name, given as a `String`.
+
+In multiparameter settings, one should define one `ParameterDistribution` per parameter, and then concatenate these either in the constructor or with `combine_distributions`. This is illustrated below and in the [Multidimensional example](@ref).
+
+### General constructor
```

Comment:
Do you think we could move the General constructor below - so the recommended constructor comes up first?